### ISSUE TITLE
Marty developer tools error

### ISSUE THREAD

**Issue by marbemac** [jhollingworth/marty#109]
```
Failed to execute 'postMessage' on 'Window': An object could not be cloned.
```

Booting up https://github.com/jhollingworth/marty-chat-example, I get the error pictured when the marty developer extension is enabled. I'm running node v0.10.6 and this is in Chrome 40.0.2214.94.

![screen shot 2015-02-04 at 11 54 30 pm](https://cloud.githubusercontent.com/assets/847542/6056705/9815517c-acc9-11e4-84d5-d8d858d39751.png)

**Comment by rottmann** [comment]
Remove line `window.Marty = Marty; // For Marty Developer Tools`
The docs say to set it, but i cant see no difference in the dev tools.

**Comment by jhollingworth** [comment]
Hey, this is a known problem. I'm hopefully going to get to it in the next few days

**Comment by dariocravero** [comment]
Dup jhollingworth/marty-devtools#4

**Comment by jhollingworth** [comment]
I've just published Marty v0.8.11 resolves this issue. The latest version of Marty DevTools (v0.8.11) should be processed in the hour which will make sure this doesn't happen again. Sorry for the delay in fixing this bug!

**Comment by marbemac** [comment]
Thanks @jhollingworth!
